### PR TITLE
Implement new debug UI

### PR DESCRIPTION
**Pull request by louneskmt**
This PR adds an option to execute the debug script via the Umbrel's dashboard. It will allow users to share with us the result of the debug script without SSHing into the Pi. 

Related PRs (depends on):
- https://github.com/getumbrel/umbrel-manager/pull/66
- https://github.com/getumbrel/umbrel/pull/391

image
image

### CODE REVIEW

**Comment by AaronDewes** [comment]
Amazing! I think @mayankchhabra needs to redesign it, but great idea & implementation! This will make debugging much easier

File: src/App.vue [hunk 1]
```diff
@@ -23,10 +23,33 @@
       </shutdown>
       <loading v-else-if="loading" :progress="loadingProgress">
         <small class="text-muted w-75 text-center">{{ loadingText }}</small>
+        <div v-if="showDebugOption">
+          <small class="text-muted w-75 text-center">If the loading is stuck, click here to debug : </small>
```
Comment (AaronDewes) [inline src/App.vue:27]:
```suggestion
          <small class="text-muted w-75 text-center">If the loading is stuck, click here to debug: </small>
```

There shouldn't be a space here: http://www.sussex.ac.uk/informatics/punctuation/colonandsemi/colon

**Review by AaronDewes — APPROVED** [review]
Only a minor change

File: src/App.vue (hunk 1 above)
Comment (louneskmt) [reply src/App.vue:27]: Sorry, french typo haha

File: src/App.vue [hunk 2]
```diff
@@ -23,10 +23,33 @@
...
+        @cancel="debugPrompt"
+        @ok="clearDebugInterval"
+      >
+        <div v-if="this.loadingDebug">
+          <p>Processing...</p>
+        </div>
+        <div v-else>
+            <p>Please share the following links with a description of your problem in the <a href="https://t.me/getumbrel">Umbrel Telegram group</a> so we can help you.</p>
+            <input-copy class="mb-1" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
+            <input-copy size="sm" auto-width :value="this.debugResult.linkDmesg"></input-copy>
```
Comment (AaronDewes) [inline src/App.vue:50]:
```suggestion
            <p>Please share the following link with a description of your problem in the <a href="https://t.me/getumbrel">Umbrel Telegram group</a> so we can help you.</p>
            <input-copy class="mb-1" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
```

**Review by AaronDewes — CHANGES_REQUESTED** [review]
You can reduce this to one link.

File: src/App.vue (hunk 2 above)
Comment (louneskmt) [reply src/App.vue]: Resolved in 676d3ed.

File: src/views/Settings.vue [hunk 3]
```diff
@@ -280,6 +280,32 @@
...
+                cancel-title="Run again"
+                @cancel="debugPrompt"
+                @ok="clearDebugInterval"
+              >
+                <div v-if="this.loadingDebug">
+                  <p>Processing...</p>
+                </div>
+                <div v-else>
+                    <p>Please share the following links with a description of your problem in the <a href="https://t.me/getumbrel">Umbrel Telegram group</a> so we can help you.</p>
+                    <input-copy class="mb-1" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
```
Comment (AaronDewes) [inline src/views/Settings.vue:304]: Can you center this? The following change could do it.

```suggestion
                    <input-copy class="mb-1 mx-auto" size="sm" auto-width :value="this.debugResult.linkDebug"></input-copy>
```

**Review by AaronDewes — APPROVED** [review]

**Pull request merged**